packages/sw-broadcast-cache-update/src/index.js

```diff
@@ -57,13 +58,15 @@
  * @module sw-broadcast-cache-update
  */
 
-import Plugin from './lib/plugin';
+import BroadcastCacheUpdate from './lib/broadcast-cache-update';
+import BroadcastCacheUpdatePlugin from './lib/broadcast-cache-update-plugin';
 import broadcastUpdate from './lib/broadcast-update';
 import {cacheUpdatedMessageType} from './lib/constants';
 import responsesAreSame from './lib/responses-are-same';
 
 export {
-  Plugin,
+  BroadcastCacheUpdate,
+  BroadcastCacheUpdatePlugin,
```

Comment:
+1. I didn't know what else to go with, though.

I'll open an issue to discuss possible changes.